test/integration/manifests/cns/daemonset-windows.yaml

```diff
@@ -71,9 +71,7 @@ spec:
             - name: log
               mountPath: /k/azurecns
             - name: cns-config
-              mountPath: /etc/azure-cns
-            - name: cni-conflist
-              mountPath: /k/azurecni/netconf
+              mountPath: /etc/azure-cns # // TODO: add windows cni conflist when ready
```

Comment:
Will this be added via code ?

Reply:
Yes. All of these values are overwritten by the test infrastructure. 

Also, we will be moving away from using these yamls for our testing infrastructure soon.